### PR TITLE
Fix #33

### LINKED ISSUE #33: `Property not found in...` errors not reported
If I have a function

``` js
function foo(bar: {baz: string}) {...}
```

And I call it with

``` js
foo(1234);
```

An error is reported as expected (due to the mismatched type). But if I call it with

``` js
foo({xyz: 'abc'});
```

An error is reported by Flow but not by this plug-in. Seems that the code filtering out issues is incorrect: https://github.com/amilajack/eslint-plugin-flowtype-errors/blob/master/src/collect.js#L76 as `filepath` and `message[0].path` don't match in this case.

### PR DESCRIPTION
**Pull request by jdmota**
I hope this is the correct fix.

### CODE REVIEW

**Comment by jdmota** [comment]
@amilajack I think it's ready for review.

File: src/collect.js [hunk 1]
```diff
@@ -5,6 +5,7 @@
  * https://github.com/facebook/nuclide/blob/master/pkg/nuclide-flow-rpc/lib/FlowRoot.js
  * https://github.com/ptmt/tryflow/blob/gh-pages/js/worker.js
  */
+import pathModule from 'path';
```
Comment (amilajack) [inline src/collect.js]: Can we use `path` instead of `pathModule`?

Comment (amilajack) [inline src/collect.js]: Ahh never mind. `path` is already defined.

Comment (jdmota) [inline src/collect.js]: I did it at first, but eslint complained: `path is already declared in the upper scope.  no-shadow`

**Comment by amilajack** [comment]
Thanks for this! I'll release in a preminor so that it can be tested

**Comment by jdmota** [comment]
No problem! :)

I'll test it as well when the release is done.

**Comment by amilajack** [comment]
Just released `eslint-plugin-flowtype-errors@2.0.2-0` 🎉

**Comment by jdmota** [comment]
It works for me!

**Comment by amilajack** [comment]
I'll test this out later today and publish a minor of everything works out for me. thanks for testing!

**Comment by amilajack** [comment]
@jdmota just published the patch version for this! Thanks again!

**Pull request merged**